### DISCUSSION TITLE
Allow Chatterino Nightly auto-updating

### DISCUSSION THREAD

**Discussion by Infinitay** [Chatterino/chatterino2#2787] [Ideas]
Allow Chatterino Nightly users to auto update.  Having enabled "Receive beta updates" in settings, I noticed that my Nightly Chatterino never updated. I suppose it's intentional though seeing as there was a commit (7cbbd72) that disabled Nightly updates. I was wondering if we could get this feature back, or was it removed for a specific reason?

**Comment by Felanbird** [comment]
> I suppose it's intentional though seeing as there was a commit ([7cbbd72](https://github.com/Chatterino/chatterino2/commit/7cbbd72e7fa49803070ded44266bdc6e8b12e808)) that disabled Nightly updates. 

The reason for this commit was to remove the arrow offering the ability to update for nightly builds, as the updater does not work for them, see #1310.

IIRC "receive beta updates" are not synonymous with the newest commits that nightly gets, I believe they still have to be manually pushed on @fourtf 's side.

Since I'm unable to find any other threads suggesting this, you can probably leave it open until somebody else who knows the coding aspect reason for why nightly doesn't update, has commented positively or negatively towards it.

**Comment by ALazyMeme** [comment]
`Receive beta updates` relates to beta builds, not nightly builds
I like the idea of this, but nightly releases on **EVERY** commit. We should time nightly released to every night UTC as opposed to on every commit.

Unless someone has a better option, I think auto-updating nightly would be nice.

**Comment by jupjohn** [comment]
If we create an update channel for nightly it would be quite useful for #2641
